sopel/modules/units.py

```diff
@@ -11,9 +11,11 @@
 
 import re
 
-from sopel.module import commands, example, NOLIMIT
+from sopel import plugin
 
 
+PLUGIN_OUTPUT_PREFIX = '[unit] '
```

Comment:
```suggestion
PLUGIN_OUTPUT_PREFIX = '[units] '
```

Is this the only nemesi**S** in this PR? Pretty amazing, tbh.

Reply:
lol